utils/options.py

```diff
@@ -0,0 +1,82 @@
+                         default=None, type=float,
+                         help="absolute: timeout for test runs in "
+                         "seconds. relative: timeout is [val] "
+                         "seconds longer than golden runtime. "
+                         "dynamic: timeout is [val] seconds longer "
+                         "than most recent successful test. "
+                         "(default: absolute. When timeout is "
+                         "unspecified, default is "
+                         "1.5 * golden runtime.)")
+    timeout_group = aparser.add_mutually_exclusive_group()
+    timeout_group.add_argument("--rel", action="store_true",
+                               dest="timeout_relative", default=False,
+                               help="timeouts are relative to \
+                                test time of input file")
+    timeout_group.add_argument("--dyn", action="store_true",
+                               dest="timeout_dynamic", default=False,
+                               help="timeouts are relative to the runtime of "
+                               "the most recent successful test")
+    aparser.add_argument("--round", dest="roundtime", metavar="val",
+                         default=None, type=float,
+                         help="approximate time limit for testing "
+                         "rounds in seconds")
+    aparser.add_argument("-v", action="count", dest="verbosity", default=0,
+                         help="increase verbosity")
+    aparser.add_argument("--match-err", dest="match_err",
+                         default=None,
+                         help="match string in stderr to identify "
+                         "failing input (default: stderr output)")
+    aparser.add_argument("--match-err-cc", dest="match_err_cc",
+                         default=None,
+                         help="match string to identify failing input "
+                         "for cross check command (default: "
+                         "stderr output)")
+    aparser.add_argument("--match-out", dest="match_out",
+                         default=None,
+                         help="match string in stdout to identify "
+                         "failing input (default: stdout output)")
+    aparser.add_argument("--match-out-cc", dest="match_out_cc",
+                         default=None,
+                         help="match string to identify failing input "
+                         "for cross check command (default: "
+                         "stdout output)")
+    aparser.add_argument("--parser-test", action="store_true",
+                         dest="parser_test", default=False,
+                         help="run ddSMT in parser test mode "
+                         "(parse only, does not require command "
+                         "argument)")
+    aparser.add_argument("--version", action="version",
+                         version=__version__)
```

Comment:
Looks like this was from an older commit and is not what is removed in ddsmt.py. I removed several options and changed the help text in my last commit.